tic_tac_toe/game.py

```diff
@@ -24,7 +28,18 @@ def _position_is_valid(position):
 
     Returns True if given position is valid, False otherwise.
     """
-    pass
+    valid_positions = [
+            (0,0), (0,1), (0,2),
+            (1,0), (1,1), (1,2),
+            (2,0), (2,1), (2,2),
+        ]
+    
+    for p in valid_positions:
```

Comment:
I think having a list of valid positions is going against the spirit of what we're supposed to be doing here, there are better more programmatic ways to accomplish this. Think of checking to make sure each of the two numbers in the tuple are valid. (ie. in all the valid positions the co-ordinates are `in (0, 1, 2)` or `in range(3)`)

That being said, even if you're sticking with using the list there's a better way to check it. A simple `if position in valid_positions:` accomplishes the same thing as your for loop here.